egs/thchs30/s5/path.sh

```diff
@@ -0,0 +1,18 @@
+#Change the following settings to match your computing env,
+#so that appropriate (CPU/GPU) Kaldi commands are visible
+
+export KALDI_ROOT=/work3/zxw/kaldi-2016-1-21
+
+if [ -z `which nvidia-smi 2>/dev/null` ];then
+#if nvidia_smi; then
+    src=src_nocuda
+    echo "using no-cuda kaldi src: $KALDI_ROOT src"
+else
+    src=src_cuda
+    echo "using cuda kaldi src: $KALDI_ROOT src"
+fi
+
```

Comment:
I agree with Yenda- you should remove all of this.
Things that are specific to a person's local system should in general never appear in scripts like these, unless they are things like directory names that you could never have a suitable default value form.